internal/core/services/events_forwarder/metrics.go

```diff
@@ -74,18 +74,18 @@ var (
 	})
 )
 
-func reportRoomEventForwardingSuccess(schedulerName string) {
+func reportRoomEventForwardingSuccess(game, schedulerName string) {
 	successRoomEventForwardingMetric.WithLabelValues(schedulerName).Inc()
```

Comment:
Did you forgot to put game as a label? xD
![image](https://user-images.githubusercontent.com/25515954/160130158-2db742aa-68ba-4c0e-9d79-1fcb740191dd.png)

Reply:
YES!